### ISSUE TITLE
Cannot use IPython at all from Spyder 2.2.1 in Ubuntu 12.04 LTS

### ISSUE THREAD

**Issue by spyder-issues-migrator** [spyder-ide/spyder#1490]
_From [miguelba...@gmail.com](https://code.google.com/u/106675673014048586124/) on 2013-07-19T06:01:58Z_

Spyder Version:  2.2.1
Python Version:  2.7.3
IPython Version: 0.12.1
Qt Version:      4.8.1, PyQt4 (API v1) 4.9.1 on Linux (Ubuntu 12.04) 

What steps will reproduce the problem?

 1. Start Spyder
2. Right-click on Console Window, or open the Interpreters menu
3. Option to launch IPython interpreter is absent What is the expected output? What do you see instead? Expected: Right-clicking on Console window should provide options for:
"Open a Python interpreter"
"Open an IPython interpreter"
(among others)
What I see instead: option for opening IPython is not present at all. 

Please provide any additional information below

. I have also searched through the Preferences window and I see no options for IPython at all.
In the "Interpreters" menu, there is only one option -- "Open a Python interpreter".
The path to my Python executable is correct in the Preferences window.
IPython works fine if I launch it from outside Spyder. As stated above, I have IPython 0.12.1.
This was working fine on Spyder 2.1.9. I upgraded to Spyder 2.2.1 using "pip install -U spyder". At that time, no error messages were displayed.

_Original issue: http://code.google.com/p/spyderlib/issues/detail?id=1490_

**Comment by spyder-issues-migrator** [comment]
_From [ccordoba12](https://code.google.com/u/ccordoba12/) on 2013-08-09T12:35:10Z_

The problem is very simple: Spyder 2.2 is compatible only with IPython 0.13 and *only* with that version.

I'm planning to make it work also with the new IPython 1.0.

**Status:** WontFix

**Comment by spyder-issues-migrator** [comment]
_From [miguelba...@gmail.com](https://code.google.com/u/106675673014048586124/) on 2013-08-12T02:45:47Z_

Thanks for your reply.

I have now installed iPython 0.13.2 through
sudo pip uninstall ipython
sudo pip install ipython==0.13.2

I also checked that iPython is correctly installed, by running "ipython" from the terminal. However, I am unable to launch Spyder. From a different issue I got the idea of running "ipython qtconsole" from the terminal; this gets me an error stating that I need something called "pyzmq". I installed that through
sudo pip install pyzmq

But after this I am still unable to launch "ipython qtconsole", and Spyder also doesn't launch. Is there a specific version of pyzmq required?
